.github/actions/release/action.yml

```diff
@@ -1,59 +1,189 @@
-          TYPE=Hotfix
+    - name: Create GITHUB releases from new release tags
+      run: |
+        # Install gh cli
+        echo "Installing GH CLI."
+        if ! command -v gh &> /dev/null; then
+          echo "Installing GH CLI."
+          type -p curl >/dev/null || (sudo apt update && sudo apt install curl -y)
+          curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg
+          sudo chmod go+r /usr/share/keyrings/githubcli-archive-keyring.gpg
+          echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+          sudo apt update
+          sudo apt install gh -y
         else
-          TYPE=Release
+          echo "GH CLI is already installed."
         fi
 
-        VERSION_DATA="{\"archived\":false,\"releaseDate\":\"$(date +%Y-%m-%d)\",\"name\":\"centreon-collect-$NEW_VERSION\",\"description\":\"$TYPE:$RELEASE_ID\",\"projectId\":${{ inputs.jira_project_id }},\"released\":false}"
+        # Create GITHUB release for each release components
+        echo "Creating GITHUB releases."
+        for TAG in ${NEW_STABLE_TAGS[@]}; do
+          echo "::notice::Creating release with title $TAG for tag $TAG."
+          gh release create $TAG --target "${{ inputs.github_ref_name }} --title "$TAG" --verify-tag
+        done
+      shell: bash
+
+    - name: Create stable JIRA versions from new release tags
+      run: |
+        set -eu
+
+        # Call JIRA to provide new jira versions to create
+        # Webhook url
+        JIRA_INCOMING_WEBHOOK="${{ inputs.jira_webhook_url }}"
+
+        # Rebuild NEW_STABLE_TAGS as an array
+        for i in ${NEW_STABLE_TAGS[@]}; do
+          NEW_RELEASE_TAGS+=("$i")
+        done
+
+        # Create new JIRA versions (old way of doing it)
+        # TODO: add a future capacity to determine wether the release is hotfix or standard (using TYPE)
+        # OR: rely on jira automation to do it (less hassle on github side, and jira knows jira best)
+
+        # Create JIRA version for each released component
+        echo "Creating JIRA releases."
+        for TAG in ${NEW_RELEASE_TAGS[@]}; do
+          echo "::notice::Creating JIRA release $TAG based on git release tag $TAG."
+          JIRA_VERSION_DATA="{\"archived\":false,\"releaseDate\":\"$(date +%Y-%m-%d)\",\"name\":\"$TAG\",\"projectId\":${{ inputs.jira_project_id }},\"released\":false}"
+          echo "Sending to JIRA release: $JIRA_VERSION_DATA"
+          # curl --fail --request POST \
+            --url 'https://centreon.atlassian.net/rest/api/3/version' \
+            --header 'Authorization: Basic ${{ inputs.jira_token }}' \
+            --header 'Accept: application/json' \
+            --header 'Content-Type: application/json' \
+            --data ''$VERSION_DATA''
```

Comment:
Why not using double quotes instead ?

Reply:
I m not really sure about the choice here since I reused the request without changing it much, since it is already used in the current "per component" release workflow
I ll check if I can make it less quoted.

Reply:
this part was remade using jq to build the json from scratch (which is easier to maintaint that raw string with bash)